src/services/identity/user/user-management/modules/UserForm.vue

```diff
@@ -312,9 +357,72 @@ export default {
             }));
         };
 
-        (async () => {
-            await Promise.all([initAuthTypeList(), getRoleList()]);
-        })();
+        const setExternalMenuItems = (users) => {
+            state.externalItems = [];
+            users.forEach((user) => {
+                const singleItem = {
+                    name: user.user_id,
+                    label: user.name ? `${user.user_id} (${user.name})` : user.user_id,
+                    disabled: false,
+                };
+                if (Object.keys(state.users).includes(user.user_id)) {
+                    singleItem.disabled = true;
+                }
```

Comment:
어차피 state.users 가 object 라면 아래처럼 바꾸면 어떨까요?
```suggestion
                if (state.users[user.user_id]) {
                    singleItem.disabled = true;
                }
```